anesthetic/samples.py

```diff
@@ -370,21 +330,6 @@ def plot_2d(self, axes=None, *args, **kwargs):
                     pos = ax.position
                     lkwargs = local_kwargs.get(pos, {})
                     lkwargs['kind'] = kind.get(pos, None)
-                    # TODO: remove this in version >= 2.1
-                    if lkwargs['kind'] == 'kde':
-                        warnings.warn(
-                            "You are using \'kde\' as a plot kind. "
-                            "\'kde_1d\' and \'kde_2d\' are the appropriate "
-                            "keywords for anesthetic. Your plots may look "
-                            "odd if you use this argument."
-                            )
-                    elif lkwargs['kind'] == 'hist':
-                        warnings.warn(
-                            "You are using \'hist\' as a plot kind. "
-                            "\'hist_1d\' and \'hist_2d\' are the appropriate "
-                            "keywords for anesthetic. Your plots may look "
-                            "odd if you use this argument."
-                            )
```

Comment:
I don't understand why these warnings should be removed?

Reply:
Agreed, these will actually still be useful for new users of anesthetic. This is different from warnings that are about changes from anesthetic 1 to anesthetic 2.

So, yes, I think we should keep them, but then they should become part of the tests. Let's address this in #342.